sensu-plugins-systemd.gemspec

```diff
@@ -1,15 +1,18 @@
+# frozen_string_literal: true
+
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 require 'date'
+require 'sensu-plugins-systemd'
 
-if RUBY_VERSION < '2.0.0'
-  require 'sensu-plugins-systemd'
-else
-  require_relative 'lib/sensu-plugins-systemd'
-end
+# if RUBY_VERSION < '2.3.0'
```

Comment:
I honestly can't remember what the change in behavior was between these versions, I'd have to look back through PRs. Should we just kill this code? Unfortunately we did not specify a min version of ruby this supports. This was one that was a community transfer and I missed adding that to the gemspec [here](https://github.com/sensu-plugins/sensu-plugins-systemd/commit/48b6b1cac2021ec19849e80aaf559c885a71c83e) would you mind adding that?